src/makeCalculator.js

```diff
@@ -38,6 +38,39 @@
  */
 function makeCalculator() {
   // write code here
+  const object = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value) {
+      this.result += value;
+    },
+
+    subtract(value) {
+      this.result -= value;
+    },
+
+    multiply(value) {
+      this.result *= value;
+    },
+
+    divide(value) {
+      this.result /= value;
+    },
+
+    operate(callback, value) {
+      callback.bind(this)(value);
```

Comment:
It's better to pass `this.result` inside callback function as an argument instead of using bind

Reply:
fixed